pyproject.toml

```
@@ -34,7 +34,10 @@ dynamic = ["version"]
 
 [project.optional-dependencies]
 dev = [
-    "build"
+    "build",
+    "pylint<=2.16.4",
```

```suggestion
    "pylint",
```
I think this constraint isn't required anymore? Did you copy and old state of the integreat config?